components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,221 @@
+        <HeroCard
+          suit={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.left
+              .suit
+          }
+          rank={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.left
+              .rank
+          }
+        />
+        <HeroCard
+          suit={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.right
+              .suit
+          }
+          rank={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.right
+              .rank
+          }
+        />
+      </HeroHand>
+
+      <PostTitle>{post.title}</PostTitle>
+
+      <Attributes>
+        <Attribute>
+          <Likes>
+            <ThumbsUpIcon />
+            {post.likes}
+          </Likes>
+        </Attribute>
+
+        <Attribute>
+          <span>Play at</span>
+          <span>
+            {getPositionByPlayerAndIndex(
+              post.gameSituation.playerLength,
+              post.gameSituation.heroIndex
+            )}
+          </span>
+        </Attribute>
+
+        <Attribute>
+          <span>Ended at</span>
+          <span>{gameEndedAt}</span>
+        </Attribute>
+
+        <Attribute>
+          <span>Final Pot</span>
+          <span>{`${getStringWithSIMetricSuffix(
+            calculateFinalPot(post.gameSituation)
+          )} BB`}</span>
+        </Attribute>
+
+        <Attribute>
+          <span>{showLastUpdate ? "Last Update" : "Posted"}</span>
+          <span>
+            <DateUnit>
+              {showLastUpdate
+                ? `${getRelativeDateString(post.lastUpdatedAt)} ago`
+                : `${getRelativeDateString(post.createdAt)} ago`}
+            </DateUnit>
+            <DateUnitInMobile>
+              {showLastUpdate
+                ? `${getRelativeShortDateString(post.lastUpdatedAt)} ago`
+                : `${getRelativeShortDateString(post.createdAt)} ago`}
+            </DateUnitInMobile>
+          </span>
+        </Attribute>
+      </Attributes>
+    </Root>
+  );
+}
+
+const Root = styled(Card)`
+  display: grid;
+  grid-template-columns: minmax(15%, 100px) 1fr;
+  grid-template-areas:
+    "playing-cards title"
+    "attributes attributes";
+  grid-gap: 12px;
+`;
+
+const HeroHand = styled.div`
+  grid-area: playing-cards;
+  position: relative;
+  width: 100%;
+
+  &: before {
+    content: "";
+    display: block;
+    padding-top: 100%;
+    background-color: #f5f6f7;
+    border-radius: 4px 0 4px 0;
+  }
+`;
+
+const HeroHandBackground = styled.div`
+  padding-top: 8px;
+  position: absolute;
+`;
+
+const HeroCard = styled(PlayingCard)`
+  position: absolute;
+  width: 40%;
+  top: 15%;
+
+  & div {
+    background-color: #fff;
+  }
+
+  &:first-of-type {
+    left 16%;
+    transform: rotate(-15deg);
+  }
+
+  &:last-of-type {
+    right 16%;
+    transform: rotate(15deg);
+  }
+`;
+
+const PostTitle = styled.h2`
+  grid-area: title;
+  font-size: 1.3em;
+  font-color: #293845;
```

Comment:
Is this intentionally new color? Or can you use colors other component uses? (e.b. [`Button`](https://github.com/agnosticful/chipstackoverflow-web/blob/master/components/Button/Button.tsx))